.drone.yml

```diff
@@ -16,6 +16,24 @@ steps:
     commands:
       - mvn -s ./timecard_settings.xml clean package
 
+  - name: sonar
+    image: maven:3.8.3-openjdk-17
+    depends_on:
+      - build
+    environment:
+      ARTIFACTORY_TOKEN:
+        from_secret: artifactory_token
```

Comment:
This was required. maven sonar scan was pulling jpa rest dependency which only lives on artefactory private repo. I guess the jparest project didn't have this requirement because all its dependencies are available on maven public repo